tests/data/many-types/input_stub.cpp

```diff
@@ -420,29 +420,32 @@ void org::gdbus::codegen::glibmm::Test::on_method_call(
         parameters.get_child(base_Param1, 0);
         std::map<Glib::ustring,Glib::VariantBase> p_Param1 = base_Param1.get();
 
+        MethodInvocation method_invocation(invocation);
         TestStringVariantDict(
             (p_Param1),
-            MethodInvocation(invocation));
+            method_invocation);
     }
 
     if (method_name.compare("TestStringStringDict") == 0) {
         Glib::Variant<std::map<Glib::ustring,Glib::ustring>> base_Param1;
         parameters.get_child(base_Param1, 0);
         std::map<Glib::ustring,Glib::ustring> p_Param1 = base_Param1.get();
 
+        MethodInvocation method_invocation(invocation);
         TestStringStringDict(
             (p_Param1),
-            MethodInvocation(invocation));
+            method_invocation);
     }
 
     if (method_name.compare("TestUintIntDict") == 0) {
         Glib::Variant<std::map<guint32,gint32>> base_Param1;
         parameters.get_child(base_Param1, 0);
         std::map<guint32,gint32> p_Param1 = base_Param1.get();
 
+        MethodInvocation method_invocation(invocation);
```

Comment:
`method_invocation{invocation};`

Reply:
_In Zoidberg voice_: The box says, **NO**!

(Do not see any other use of this, and I know it is just your preference. :) If you are serious, I can ofcourse change it if it is a style we want to use.)

Reply:
Actually i was joking knowing that @mardy would like discussion :)
On the other hand, although Scott Meyers says choose one style and be consistent there are other sources saying use {} as well. For instance C++ core guidelines; https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es23-prefer-the--initializer-syntax

Reply:
Listen to the box. ;)